public/app/features/alerting/unified/components/rules/RulesTable.tsx

```diff
@@ -137,8 +136,18 @@ function useColumns(showSummaryColumn: boolean, showGroupColumn: boolean) {
         // eslint-disable-next-line react/display-name
         renderCell: ({ data: rule }) => {
           const { namespace, group } = rule;
-          const { rulesSource } = namespace;
-          return isCloudRulesSource(rulesSource) ? `${namespace.name} > ${group.name}` : namespace.name;
+
+          // ungrouped rules are rules that are in the "default" group name
+          const isUngrouped = group.name === 'default';
```

Comment:
This code looks almost the same as https://github.com/grafana/grafana/blob/81a5df591caa1fe0097b34e7b142297b3e50f750/public/app/features/alerting/unified/components/rules/RulesGroup.tsx#L122:L129
Maybe we can extract a common component for them?

Reply:
Good idea!